### PR TITLE
The conformer for URL was broken

### PR DESCRIPTION
**Pull request by acron0**

### CODE REVIEW

File: src/kixi/spec/conformers.clj [hunk 1]
```diff
@@ -289,16 +289,16 @@
...
-    (s/conformer url? identity)
-    (fn []
-      (gen/fmap fmt-url
-                (gen/tuple
-                 (gen/boolean)
-                 (gen/vector (gen/such-that #(< 1 (count %))  (gen/string-alphanumeric)) 2 5)
-                 (gen/vector (gen/such-that #(< 1 (count %))  (gen/string-alphanumeric)) 2 5)
-                 (gen/one-of [(gen/such-that #(< 1 (count %)) (gen/string-alphanumeric))
-                              (gen/return nil)]))))))
+  (let [at-least-one #(< 1 (count %))]
```
Comment (mantree) [inline src/kixi/spec/conformers.clj]: could you just use 'seq' for this?

Comment (mantree) [reply src/kixi/spec/conformers.clj]: not-empty might be clearer

Comment (acron0) [reply src/kixi/spec/conformers.clj]: Yeah, good point. I'll use `(not (clojure.string/blank? %))`

File: src/kixi/spec/conformers.clj [hunk 2]
```diff
@@ -289,16 +289,16 @@
...
-                 (gen/vector (gen/such-that #(< 1 (count %))  (gen/string-alphanumeric)) 2 5)
-                 (gen/one-of [(gen/such-that #(< 1 (count %)) (gen/string-alphanumeric))
-                              (gen/return nil)]))))))
+  (let [not-blank #(not (clojure.string/blank? %))]
+    (s/with-gen
+      (s/conformer -url? identity)
+      #(gen/fmap fmt-url
+                 (gen/tuple
+                  (gen/boolean)
+                  (gen/vector (gen/such-that not-blank  (gen/string-alphanumeric)) 2 5)
```
Comment (mantree) [inline src/kixi/spec/conformers.clj]: just straight up "not-empty":
kixi.event-fixer> (not-empty nil)
nil
kixi.event-fixer> (not-empty "")
nil
kixi.event-fixer> (not-empty "a")
"a"

Comment (mantree) [reply src/kixi/spec/conformers.clj]: O, your worried about " ", but this won't stop "a b" will it?

Comment (acron0) [reply src/kixi/spec/conformers.clj]: Doesn't account for whitespace

**Pull request merged**